### ISSUE TITLE
TSV parser object needed (with json schema)

### ISSUE THREAD

**Issue by luiskuhn** [qbicsoftware/etl-scripts#73]
> I will make sure to create this documentation and the suggested TSV parser object, but I would suggest making that another card, since that directly relates to the defined metadata model for each imaging modality (which I am still working on), input validation and controlled vocabulary. There I think we need to go the same route as for the nanopore data, but I see that separate from building/testing the functionality to ingest metadata to the omero server as key-value pairs.

Ok, but then please create an issue in this repo that this is not forgotten. This needs to be done before it is roled-out in production.

_Originally posted by @sven1103 in https://github.com/qbicsoftware/etl-scripts/issues/71#issuecomment-786523632_
